src/main.py

```diff
@@ -1,9 +1,21 @@
 import datetime
+
+import inspect
+from dataclasses import asdict
+import os
```

Comment:
- Wrong import order (imports within one grup should be in alphabetic order, and `import ...` should be before `from ... import`)
- Unnecessary newline after datetime import
- inspect and asdict import statements are unused. Also generate_invoice_xml and process_form which are in the next few lines

https://peps.python.org/pep-0008/#imports